docs/self-hosted-apps/dashboards.md

```diff
@@ -0,0 +1,30 @@
+# Self Hosted Dashboards
+
```

Comment:
Add a short description about what you mean by "dashboards". Is [netdata](https://www.netdata.cloud/) for e.g. a dashboard?